hoot-core/src/main/cpp/hoot/core/algorithms/changeset/ChangesetTaskGridReplacer.cpp

```diff
@@ -214,7 +214,6 @@ OsmMapPtr ChangesetTaskGridReplacer::_getNodeDensityTaskGridInput()
     LOG_STATUS(
       "Preparing input data for task grid cell calculation from: ..." <<
       _replacementUrl.right(25) << "...");
-    assert(HootApiDbReader::isSupported(_replacementUrl));
```

Comment:
`HootApiDbReader::isSupported` isn't static and cannot be compiled like this in `DEBUG` mode.  `RELEASE` just ignores anything inside of it and compiles just fine.